examples/gallery/embellishments/solar.py

```diff
@@ -0,0 +1,46 @@
+"""
+Day-night terminator line and twilights
+---------------------------------------
+
+Use :meth:`pygmt.Figure.solar` to plot the day-night terminator line, 
+and civil, nautical, astronomical twilights.
```

Comment:
Best if we can expand the description a bit more, so that people can understand the different stages of twilight (I actually didn't know about it until now)!

```suggestion
Use :meth:`pygmt.Figure.solar` to show the different transition stages between daytime
and nightime. The parameter ``terminator`` is used to set the twilight stage, and can be
either 'day-night' (brightest), 'civil', 'nautical', or 'astronomical' (darkest). Refer
to https://en.wikipedia.org/wiki/Twilight for more information.
```